scripts/qiita

```diff
@@ -425,18 +424,5 @@ def start(port):
     tornado.ioloop.IOLoop.instance().start()
 
 
-# #############################################################################
-# WARE COMMANDS
-# #############################################################################
-
-@ware.command()
-@click.argument('sl_out_dir', required=True,
-                type=click.Path(resolve_path=True, readable=True, exists=True,
-                                file_okay=False))
-def generate_demux(sl_out_dir):
```

Comment:
I feel like the CLIs have proved useful to add/process studies in bulk,
and given that I recently used this tool for one study myself, the new
issue is here:
https://github.com/biocore/qiita/issues/1563

On (Dec-04-15|13:54), Jose Navas wrote:
> > @@ -425,18 +424,5 @@ def start(port):
> >      tornado.ioloop.IOLoop.instance().start()
> >  
> >  
> > -# #############################################################################
> > -# WARE COMMANDS
> > -# #############################################################################
> > -
> > -@ware.command()
> > -@click.argument('sl_out_dir', required=True,
> > -                type=click.Path(resolve_path=True, readable=True, exists=True,
> > -                                file_okay=False))
> > -def generate_demux(sl_out_dir):
> 
> generate demux is part of the plugin. I'm unsure if we want to add CLI to the plugins... I would suggest adding an issue to see what we can do. Note that this functionality is available anyways through ipython, by importing the generate_demux_file func from the plugin
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1558/files#r46737064